plenum/server/node.py

```diff
@@ -1764,8 +1764,8 @@ def preLedgerCatchUp(self, ledger_id):
 
         # revert uncommitted txns and state for unordered requests
         r = self.master_replica.revert_unordered_batches()
-        logger.info('{} reverted {} batches before starting catch up for '
-                    'ledger {}'.format(self, r, ledger_id))
+        logger.trace('{} reverted {} batches before starting catch up for '
```

Comment:
I think this is quite important, and not so common (only called before catchup, and each catchup requires attention).
I think it should have DEBUG level.